2022-07-31 15:34:57,509 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-07-31 15:34:57,509 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from /Users/magi-nerv/opt/anaconda3/lib/python3.9/site-packages/instabot/bot/bot.py
2022-07-31 15:34:57,511 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-07-31 15:34:58,929 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2022-07-31 15:34:58,930 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2022-07-31 15:34:58,930 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2022-07-31 15:35:00,463 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2022-07-31 15:35:00,464 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2022-07-31 15:35:00,464 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
